matchbox_demo/README.md

```diff
@@ -22,7 +22,7 @@ cargo install wasm-server-runner
 ## Build and serve
 
 ```
-cargo run --target wasm32-unknown-unknown
+cargo run --release --target wasm32-unknown-unknown
```

Comment:
This is to get smaller builds, right? Or because you needed the performance?

There are a couple of minor downsides to using release builds, like for instance some validation layers are turned off in bevy that provide quite helpful debugging messages.

Perhaps it would be better to set something in .cargo/config.toml or Cargo.toml if that's possible.

Not a big deal, though.

Reply:
I just put it there to squash build size. This change is absolutely not necessary and should be removed if not 100% upsides. I will remove it.